portfolio/src/main/webapp/miscPage.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Create close button for expanding info box.
+ */
+function createCloseButtonElement() {
+  const buttonElement = document.createElement('button');
+  buttonElement.innerText = 'Close';
+  buttonElement.onclick = function () {
+    closeInfo();
+  };
+  buttonElement.className = 'misc-button';
+  return buttonElement;
+}
+
+/**
+ * Creates a map and adds it to the page.
+ */
+function createMap() {
+  const map = new google.maps.Map(document.getElementById('map'), {
+    //center: {lat: 37.412177, lng: -121.959926},
+    center: { lat: 37.39, lng: -121.945 },
+    zoom: 13,
+  });
+  const GRTStartMarker = new google.maps.Marker({
```

Comment:
Acronyms should be put into camelcase as if they were a word. So "grtStartMarker".

https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined